hpat/distributed.py

```diff
@@ -73,10 +79,38 @@
 
 
 class DistributedPass(object):
-    """analyze program and transfrom to distributed"""
+    """The summary of the class should be here for example below is the summary line for this class
+    
+    This class analyzes program and transforms to distributed
+    """
 
     def __init__(self, func_ir, typingctx, targetctx, typemap, calltypes,
                  metadata):
+        """ Here are moer details about the init method or the class 
+       
+        The details can be multiple lines. You can mention all the supported
+        and not supported args here or you can do it in ``Parameters`` section
+        
+        Note
+        ----
+        Do not include the `self` paramter in the ``Parameters`` section
+        
+        Parameters
+        -----------
+        func_ir : str
+             description of `func_ir`.(optional) you can mention supported or not here
+        typingctx : TypeForThisParam
```

Comment:
Can we add a link to external site? I mean, need HTML link example.

Reply:
Yes you can. The same way you do it in rst file.  For an example I tried to add link to Pandas documentation and it worked.